common/src/sim_config.rs

```diff
@@ -45,10 +45,10 @@ impl SimConfig {
         let meters_to_absolute = meters_to_absolute(chunk_size, voxel_size);
         SimConfig {
             rate: x.rate.unwrap_or(10),
-            view_distance: x.view_distance.unwrap_or(75.0) * meters_to_absolute,
+            view_distance: x.view_distance.unwrap_or(90.0) * meters_to_absolute,
```

Comment:
This default should not be changed without a proportionate performance optimization.

Reply:
Changed it back to the original.